src/models/constants/ApplicationStatus.js

```diff
@@ -1,7 +1,10 @@
 const ApplicationStatus = Object.freeze({
+    "UNVERIFIED": "UNVERIFIED",
     "PENDING": "PENDING",
     "APPROVED": "APPROVED",
     "REJECTED": "REJECTED",
 });
 
 export default ApplicationStatus;
+
+export const RECOVERY_LINK_EXPIRATION = 600;
```

Comment:
What is this being used for? Also, wrong naming as this is used for confirmation

Reply:
Should be `CONFIRMATION_LINK_EXPIRATION` or something like that

Reply:
Yes the naming is completely wrong( copy paste problem) it's used as a constant of the expiration time of the token